spinn_front_end_common/utility_models/data_speed_up_packet_gatherer_machine_vertex.py

```diff
@@ -272,22 +279,57 @@ def get_local_provenance_data(self):
                                     n_lost_seq_nums))))
         return prov_items
 
-    @staticmethod
     def set_cores_for_data_extraction(
-            transceiver, extra_monitor_cores_for_router_timeout,
+            self, transceiver, extra_monitor_cores_for_router_timeout,
             placements):
+
+        # Store the last reinjection status for resetting
+        # NOTE: This assumes the status is the same on all cores
+        self._last_reinjection_status = \
+            extra_monitor_cores_for_router_timeout[0].get_reinjection_status(
+                placements, transceiver)
+
         # set time out
         extra_monitor_cores_for_router_timeout[0].set_router_time_outs(
             15, 15, transceiver, placements,
             extra_monitor_cores_for_router_timeout)
+        extra_monitor_cores_for_router_timeout[0].\
+            set_reinjection_router_emergency_timeout(
+                1, 1, transceiver, placements,
+                extra_monitor_cores_for_router_timeout)
 
-    @staticmethod
     def unset_cores_for_data_extraction(
-            transceiver, extra_monitor_cores_for_router_timeout,
+            self, transceiver, extra_monitor_cores_for_router_timeout,
```

Comment:
Same problems; excessively long name and no documentation comment on a public method

Reply:
As above...